### DISCUSSION TITLE
Navigation doesn't work

### DISCUSSION THREAD

**Discussion by liamseys** [spatie/laravel-livewire-wizard#57] [Q&A]
Hi all, 

I'm trying to add a navigation to show the process through the wizard to the end-user. However I am receiving exceptions that my methods on the StepComponents do not exist. The docs don't mention a lot about it so I wonder if I do anything anything wrong.

Best

**Comment by Ceepster14** [comment]
This sounds like the blade views you're using for your steps are incorrectly structured. Have you checked that all the elements in your step view are contained in a single parent element as per Livewire requirements?

If not, then Livewire won't handle the events correctly. Events (such as a click on a navigation link) from your StepComponent will be passed to to the parent WizardComponent instead, which could explain the missing method exceptions you're seeing.
